client/src/models.ts

```diff
@@ -84,7 +84,8 @@ export interface DraftReqI {
   request_type_cd?: string
 }
 export interface EntityI {
-  blurb?: string[] | string
+  blurbs?: string[] | string
```

Comment:
This one should be `blurb(s)?: string[] | string`.





(Kidding, but really how do you tell if is a string or an array?)

Reply:
Good question, i think with the recent refactors there isn't any instances of it being a single string, but i will confirm. 
I think i converted them all to arrays since we will have variations of the blurbs based on all the different flows 😓 
Will give this a look when i finish up the copy.

Reply:
I saw one structure where "blurbs" is a single string, but it might be hard to untangle where it's used as an array or a string so I'm OK with leaving it alone now.